### DISCUSSION TITLE
Help can not fix the following Error

### DISCUSSION THREAD

**Discussion by thecodex101** [community/community#33293] [New to GitHub]
PS G:\My Drive\Folder\GitRepoFolder> git push -u origin main
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

I can not get around this issue, am trying to do from VSCODE

can anyone help

**Comment by airtower-luna** [comment]
This error means that either your SSH public key wasn't recognized by GitHub, or you don't have one at all:

`git@github.com: Permission denied (publickey).`

First, make you've added your public key to your account. If you've done that, run the push with `GIT_SSH_COMMAND="ssh -v"` to get debug info from SSH (including which keys are tried).
